src/i18n/content/jp/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx

```diff
@@ -9,7 +9,7 @@ freshnessValidatedDate: '2023-07-19T00:00:00.000Z'
 translationType: machine
 ---
 
-import signalInsightHorizontalView from 'images/distributed-tracing_entity-signal-insight-horizontal.webp'
+import signalInsightHorizontalView from 'images/apm_screenshot-crop_distributed-tracing-in-the-apm-summary-page.webp'
```

Comment:
These get updated automatically, so we don't need to edit i18n files. I'll undo these in your PR